examples/sample-app/cleanup.sh

```diff
@@ -1,10 +1,15 @@
 #!/bin/sh
 
 echo "Killing openshift all-in-one server ..."
-pkill -x openshift
+sudo pkill -x openshift
+
+echo "Stopping all k8s docker containers on host ..."
+sudo docker ps --format='{{.Names}}' | awk 'index($NF,"k8s_")==1 { print $1 }' | xargs -l -r sudo docker stop
```

Comment:
Why are these `sudo docker` now? I feel like it's reasonable to have as a prerequisite that the user doing these actions is in the `docker` group.